acceptance/analyzer_test.go

```diff
@@ -1214,13 +1218,13 @@ func minifyMetadata(t *testing.T, path string, metadataStruct interface{}) strin
 	return string(flatMetadata)
 }
 
-func buildAuthRegistryImage(t *testing.T, repoName, context string, buildArgs ...string) (string, string) {
+func buildAuthRegistryImage(t *testing.T, repoName, context string, registry *ih.DockerRegistry, buildArgs ...string) (string, string) {
```

Comment:
```suggestion
func buildRegistryImage(t *testing.T, repoName, context string, registry *ih.DockerRegistry, buildArgs ...string) (string, string) {
```
I wonder if it still makes sense to have this function return the auth. Since we're only getting the auth one time... maybe we could just pull that out, and then we don't have to ignore the second return value every other time.

Reply:
I did the change, you can take a look and give me your thoughts